tuta-sdk/rust/sdk/Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Are the lock files inside `sdk` and `uniffi-bingen` needed? [The cargo book claims crates in a workspace share a single lock file](https://doc.rust-lang.org/cargo/reference/workspaces.html#workspaces).